seqr/views/apis/dataset_api_tests.py

```diff
@@ -15,6 +15,7 @@
 PROJECT_GUID = 'R0001_1kg'
 INDEX_NAME = 'test_index'
 SV_INDEX_NAME = 'test_new_sv_index'
+NEW_INDEX_NAME = 'test_new_index'
```

Comment:
Would it be more descriptive to rename `NEW_INDEX_NAME` to `NEW_SAMPLE_TYPE_INDEX_NAME` or a similar name else?

Reply:
done